### PR TITLE
improve grammar for 'return variable' gotcha

### PR DESCRIPTION
**Pull request by ad8-bdl**
The current version reads poorly to me.

- [ x] I have reviewed the [Contributor Guidelines][contributor].
- [ x] I have reviewed the [Code of Conduct][coc] and agree to abide by it

[contributor]: https://github.com/bats-core/bats-core/blob/master/docs/CONTRIBUTING.md
[coc]:         https://github.com/bats-core/bats-core/blob/master/docs/CODE_OF_CONDUCT.md

### CODE REVIEW

**Review by martin-schulze-vireso — CHANGES_REQUESTED** [review]

File: docs/source/gotchas.rst [hunk 1]
```diff
@@ -36,8 +36,8 @@ If you need such a feature, please let us know about your usecase.
 
 As a workaround you can use environment variables to pass parameters.
 
-Testing functions that return their results via a variable.
------------------------------------------------------------
+How can I test functions that return their results via a variable?
```
[the remark this answers is not in the document]
Comment (ad8-bdl) [reply docs/source/gotchas.rst]: sgtm, fixed

**Review by martin-schulze-vireso — APPROVED** [review]

File: docs/source/gotchas.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (martin-schulze-vireso) [reply docs/source/gotchas.rst]: sgtm?

**Comment by martin-schulze-vireso** [comment]
I had rebase to sign the commits and squashed them too.

File: docs/source/gotchas.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (ad8-bdl) [reply docs/source/gotchas.rst]: sorry, sgtm = "sounds good to me" :-)

[the remark this answers is not in the document]
Comment (martin-schulze-vireso) [reply docs/source/gotchas.rst]: Okay, thanks for the clarification.

**Pull request merged**